app/pathogen/arbovirus/wip/sections.tsx

```diff
@@ -0,0 +1,73 @@
+import { RefObject } from "react";
+
+enum RedesignedArbovirusPageSectionId {
+  MAP = 'MAP',
+  TABLE = 'TABLE',
+  VISUALIZATIONS = 'VISUALIZATIONS'
+}
+
+enum RedesignedArbovirusPageSectionUrlParam {
+  map = 'map',
+  table = 'table',
+  visualizations = 'visualizations'
+}
```

Comment:
Same logic as the separation for the visualizations, we should never delete url params so that we won't break links that are out in the wild.